.github/workflows/release-notes-issue.yml

```diff
@@ -0,0 +1,109 @@
+        with:
+          node-version-file: .nvmrc
+          cache: "pnpm"
+          cache-dependency-path: pnpm-lock.yaml
+
+      # Get the version of the client release group by reading the root package.json and set it as an output variable.
+      # This way we can use the version in subsequent steps.
+      - name: Set version output variable
+        id: setVersion
+        run: |
+          echo "VERSION=$(jq -r '.version' package.json)" >> "$GITHUB_OUTPUT"
+
+      # Install the in-repo build-tools.
+      - name: Install Fluid build tools
+        continue-on-error: true
+        run: |
+          cd build-tools
+          pnpm install --frozen-lockfile
+          pnpm run build:compile
+          # We want flub available to call, so we run npm link in the build-cli package, which creates shims that are avilable on the PATH
+          # Use npm link instead of pnpm link because it handles bins better
+          cd packages/build-cli
+          npm link
+
+      - name: Check build-tools installation
+        run: |
+          # Info for debugging
+          which flub
+          flub --help
+          flub commands
+
+      # Generates the release notes based on the changesets in the repo. Changes that don't map to known sections WILL
+      # be included.
+      - name: Generate release notes file
+        run: |
+          flub generate releaseNotes -g client -t minor --includeUnknown --out RELEASE_NOTES.md -v
+
+      # Read the release notes file that we just generated into an output variable.
+      - name: Read release notes file
+        id: relNotes
+        uses: juliangruber/read-file-action@b549046febe0fe86f8cb4f93c24e284433f9ab58 # ratchet:juliangruber/read-file-action@v1
+        with:
+          path: ./RELEASE_NOTES.md
+
+      # Read the issue intro from a data file and put it in an output variable.
+      - name: Read issue intro file
+        id: intro
+        uses: juliangruber/read-file-action@b549046febe0fe86f8cb4f93c24e284433f9ab58 # ratchet:juliangruber/read-file-action@v1
+        with:
+          path: ${{ github.workspace }}/.github/workflows/data/release-notes-issue-intro.md
+
+      # Replace the current issue body with the release notes we generated.
+      - name: Replace issue body with release notes
+        uses: julien-deramond/update-issue-body@a7fae45395cac5a23318d38f7b09a58650dfe84f # ratchet:julien-deramond/update-issue-body@v1
+        with:
+          issue-number: ${{ env.ISSUE }}
+          body: ${{ steps.relNotes.outputs.content }}
+          edit-mode: replace
+
+      # Prepend the issue intro to the issue.
+      - name: Prepend issue intro
+        uses: julien-deramond/update-issue-body@a7fae45395cac5a23318d38f7b09a58650dfe84f # ratchet:julien-deramond/update-issue-body@v1
+        with:
+          issue-number: ${{ env.ISSUE }}
+          body: ${{ steps.intro.outputs.content }}
+          edit-mode: prepend
```

Comment:
Not a blocker, just wondering if there's also an append edit-mode. Feels unusual to replace with the main body then prepend the intro, vs replace with the intro then append the body.

Reply:
I could do it that way - but it makes the diffs in the issue history a little less useful. Not a big deal. I'll rearrange the logic and append the main body.

Reply:
Oh, I see why that would be... I don't mind the prepend that much to be honest, so a comment explaining that's the reason would be good enough, IMO. I leave it up to you.